### PR TITLE
Clean the project

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,13 +0,0 @@
-Copyright (C) 2011-2015 lishid.  All rights reserved.
```
Comment (ProgrammerDan) [inline README:1]: This could be renamed to NOTICE but should not be removed.

**Comment by ProgrammerDan** [comment]
Glad to see additional interest!

File: README (hunk 1 above)
Comment (sgdc3) [inline README:1]: You're right, i haven't checked if it was included into the LICENSE file ;)

Comment (ProgrammerDan) [inline README:1]: No worries :D I added it back to the end of the README, so all is well. Thanks again for this pull!

**Pull request merged**